src/Symfony/Component/Serializer/Mapping/Loader/AnnotationLoader.php

```diff
@@ -70,25 +75,29 @@ public function loadClassMetadata(ClassMetadataInterface $classMetadata)
         foreach ($reflectionClass->getMethods() as $method) {
             if ($method->getDeclaringClass()->name === $className) {
                 foreach ($this->reader->getMethodAnnotations($method) as $groups) {
+                    if (!preg_match('/^(get|is|has|set)(.+)$/i', $method->name, $matches)) {
```

Comment:
Sometimes the getter name is simply the name of the property without any prefix. I think it would be good to support it too.